lib/node_modules/@stdlib/stats/base/minsorted/README.md

```diff
@@ -139,16 +130,10 @@ var v = minsorted.ndarray( N, x, 2, 1 );
 <!-- eslint no-undef: "error" -->
 
 ```javascript
-var Float64Array = require( '@stdlib/array/float64' );
+var linspace = require( '@stdlib/array/base/linspace' );
 var minsorted = require( '@stdlib/stats/base/minsorted' );
 
-var x;
-var i;
-
-x = new Float64Array( 10 );
-for ( i = 0; i < x.length; i++ ) {
-    x[ i ] = i - 5.0;
-}
+var x = linspace( -5, 5, 10);
```

Comment:
```suggestion
var x = linspace( -5.0, 5.0, 10);
```

In general, if inputs may/should be decimal values, add decimals. This is our conventions. Where an input argument may only be an integer, omit decimals.